docs/docusaurus/versioned_docs/version-1.5.0/lte/upgrade_1_5.md

```diff
@@ -0,0 +1,29 @@
+---
+id: version-1.5.0-upgrade_1_5
+title: Upgrade to v1.5
+hide_title: true
+original_id: upgrade_1_5
+---
+
+# Upgrade to v1.5
+
+### Repo Change
+
+In Magma Fuji (v1.5), Magma artifacts are now hosted on the new Magmacore repositories.
+Gateways migrating from older Magma releases can run the migration script to update the sources accordingly.
+
+This repository currently supports both Debian and Ubuntu OS flavors:
+
+```bash
+wget https://raw.githubusercontent.com/magma/magma/master/lte/gateway/release/upgrade_magma.sh
+chmod +x upgrade_magma.sh
+./upgrade_magma_sh
```

Comment:
can you change the file permission of this script in git and remove chmod.

Reply:
done